package.json

```diff
@@ -163,7 +163,7 @@
         "@types/redux-devtools": "3.0.47",
         "@types/redux-devtools-extension": "2.13.2",
         "cross-env": "7.0.2",
-        "cypress": "5.6.0",
+        "cypress": "6.0.0",
```

Comment:
this shouldn't be necessary anymore

Reply:
I guess you mean the resolution a few lines below this? And I'm not sure I did not test this and as it is already changed in main it doesn't even matter in this PR.

Reply:
I misunderstood you, but after the rebase from main it should not even be a change anymore.